data.tf

```diff
@@ -12,6 +12,11 @@ data "aws_iam_policy_document" "workers_assume_role_policy" {
       type        = "Service"
       identifiers = ["ec2.amazonaws.com"]
     }
+
+    principals {
+      type        = "AWS"
+      identifiers = "${var.workers_assume_role_policy_principals_aws_identifiers}"
```

Comment:
According to the variable default, this is a list. But here your value isn't inside `[]`. Is this correct?

Reply:
It works and I found `["${var.workers_assume_role_policy_principals_aws_identifiers}"]` also works. How do you think which one is better for readability? I prefer to not write redundant codes, so I wrote like it.

Reply:
Yes, inside `[]` please. It's the standard way in Terraform 0.11. Whether it's readable or makes sense is another question 😅